src/ScottPlot5/ScottPlot/Plottables/Heatmap.cs

```diff
@@ -22,27 +22,104 @@ public class Heatmap : IPlottable
         public bool IsVisible { get; set; } = true;
         public IAxes Axes { get; set; } = Axis.Axes.Default;
         public IColormap Colormap { get; set; } = new Viridis();
+
+        /// <summary>
+        /// Indicates position of the data point relative to the rectangle used to represent it.
+        /// An alignment of upper right means the rectangle will appear to the lower left of the point itself.
```

Comment:
`UpperRight` doesn't exist on the enum

Reply:
Also if we want to support having independent alignments for each dimension (which I would caution against), I'd rather put them as two separate properties than have a god enum like this:

```cs
public enum CellAlignment {
  BottomLeft
  BottomCenter
  BottomRight
  MiddleLeft,
  .
  .
  .
}
```